packages/angular/projects/vcd/plugin-builders/src/lib/base/index.ts

```diff
@@ -120,7 +134,7 @@ async function commandBuilder(
     }
 
     // Define amd lib
-    config.output.filename = 'bundle.js';
+    config.output.filename = '[name].js';
```

Comment:
The previous code change was exactly aimed at using 'bundle.js'
Why it is needed to revert it back to '[name].js'?